<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1688</width>
    <height>1110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestRunner</string>
  </property>
  <property name="toolTip">
   <string>Sends a texte message to selected computers</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidgetComputers">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>401</width>
      <height>971</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
    </property>
    <property name="tabKeyNavigation">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>1251</width>
      <height>1031</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="system_tab">
     <attribute name="title">
      <string>System</string>
     </attribute>
     <widget class="QPushButton" name="SendRunProcessBtn">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>950</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Sends a texte message to selected computers</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="text">
       <string extracomment="Sends a texte message to selected computers">Run Programme</string>
      </property>
     </widget>
     <widget class="QTreeWidget" name="treeWidgetProcess">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1081</width>
        <height>911</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="SendMessageBtn">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>950</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Sends a texte message to selected computers</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="text">
       <string extracomment="Sends a texte message to selected computers">Send Message to 
selected machine</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>950</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="RunProcessTextEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>950</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SendKillProcessBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>950</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Sends a texte message to selected computers</string>
      </property>
      <property name="toolTipDuration">
       <number>5</number>
      </property>
      <property name="text">
       <string extracomment="Sends a texte message to selected computers">Kill Selected Process</string>
      </property>
     </widget>
     <widget class="QLabel" name="MessageRecv_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Running processes of :</string>
      </property>
     </widget>
     <widget class="QLabel" name="SelectedMachineLbl">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Machine X</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="functionnal_tab">
     <attribute name="title">
      <string>Autotest</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="MessageRecv_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>1050</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Received Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="MessageRecv">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>1050</y>
      <width>621</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="buddy">
     <cstring></cstring>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
